### PR TITLE
Mapping table

### PR DESCRIPTION
**Pull request by floWetzels**
Added a table describing the mapping between `ISA` and `schema.org`/`bioschemas`, on which the profile is built.

### CODE REVIEW

File: profile/isa_ro_crate_mapping.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+|MaterialAttributeValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)|See details for properties below.|
+|ProcessParameterValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)|See details for properties below.|
+||
+|OntologySourceReference|-|dropped type|
+|Factor|-|dropped type|
+|Material|-|dropped type|
+|ProtocolParameter|-|dropped type|
+|MaterialAttribute|-|dropped type|
+
+The three types `MaterialAttributeValue`, `FactorValue` and `ProcessParameterValueValue` are all encoded as a [schema.org/PropertyValue](https://schema.org/PropertyValue).
```
Comment (HLWeil) [inline profile/isa_ro_crate_mapping.md:139]: Give this section a header. Maybe "Value triplets"?

File: profile/isa_ro_crate_mapping.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+The value field is encoded in a straight-forward manner.
+The unit is again an ontology term that has to be resolved into a URL and a name.
+
+| ISA term | schema term | Explanation |
+|----------|-------------|-------------|
+|MaterialAttributeValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)||
+|FactorValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)||
+|ProcessParameterValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)||
+|@id|@id||
+|category|name||
```
Comment (HLWeil) [inline profile/isa_ro_crate_mapping.md:151]: Add explanations, e.g. "category text", "category ur", "value text", "value uri" ...

File: profile/isa_ro_crate_mapping.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
+| ISA term | schema term | Explanation |
+|----------|-------------|-------------|
+|Investigation|Dataset||
```
Comment (HLWeil) [inline profile/isa_ro_crate_mapping.md:3]: Any way, to distinguish between types and properties? Are "`" being formatted in tables?

File: profile/isa_ro_crate_mapping.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
+| ISA term | schema term | Explanation |
```
Comment (HLWeil) [inline profile/isa_ro_crate_mapping.md:1]: Initial header for the markdown and one sentence, smt like: This is the mapping between isa json types defined in https://isa-specs.readthedocs.io/en/latest/isajson.html and isa-ro-crate-json profile defined in https://github.com/nfdi4plants/isa-ro-crate-profile/blob/main/profile/isa_ro_crate.md .

File: profile/isa_ro_crate_mapping.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+|name|name||
+|value|text||
+||
+|OntologyAnnotation|DefinedTerm||
+|annotationValue|name||
+|termSource|inDefinedTermSet||
+|termAccession|termCode||
+|comments|disambiguatingDescription|Comment object in ISA has to be encoded as a string.|
+||
+|FactorValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)|See details for properties below.|
```
Comment (HLWeil) [inline profile/isa_ro_crate_mapping.md:129]: When section below has a header, link to it?

File: profile/isa_ro_crate_mapping.md (hunk 3 above)
Comment (HLWeil) [reply profile/isa_ro_crate_mapping.md:3]: Or maybe use links to the isa json types and the schema.org types? That would give a direct visual differentiation in addition to the single empty line.

E.g.

|----------|-------------|-------------|
|[Investigation](https://isa-specs.readthedocs.io/en/latest/isajson.html#investigation-schema-json)|[Dataset](https://schema.org/Dataset)||

**Review by HLWeil — CHANGES_REQUESTED** [review]
Good first version!

I think some additional structure might help for readability.

**Comment by stuzart** [comment]
> Good first version!
> 
> I think some additional structure might help for readability.

Yeah, but I'm not sure how within the limitations of markdown. If the tables are split per type, then they can end up with different width columns, but might be worth a try

File: profile/isa_ro_crate_mapping.md (hunk 5 above)
Comment (stuzart) [reply profile/isa_ro_crate_mapping.md:129]: yeah, I think this would help.

**Review by stuzart — APPROVED** [review]
I think this is a good idea, and other than @HLWeil 's suggestions looks good.

**Comment by floWetzels** [comment]
I think I fixed all the issues listed above. Could you have another look, @HLWeil @stuzart ?

**Review by HLWeil — APPROVED** [review]
Looks good!

**Pull request merged**